src/shared/remoted_op.c

```diff
@@ -79,21 +79,27 @@ void parse_uname_string (char *uname,
         // Get os_major
         if (w_regexec("^([0-9]+)\\.*", str_tmp, 2, match)) {
             match_size = match[1].rm_eo - match[1].rm_so;
-            *os_major = malloc(match_size +1 );
+
```

Comment:
Style: Isn't better without this extra empty lines?

Reply:
Well, not sure why I did it... Fixed 😄